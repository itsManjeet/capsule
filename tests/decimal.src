println (10.00);
println (10.00 + 30.39);
println (30.95 - 40.29);
println (40.17 / 13.763);
println (10.123 == 10.123);
println (44.050 != 10.374);
println (19.45 > 405.01);
println (19.45 < 405.01);
println (19.45 <= 405.01);
println (19.45 >= 405.01);
println (not 29.04);