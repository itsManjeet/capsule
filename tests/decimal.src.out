10.000000
40.390000
-9.340000
2.918695
true
true
false
true
true
false
false
